app/components/search-facet-daterange/component.js

```diff
@@ -45,11 +46,23 @@ export default Ember.Component.extend({
 
     },
 
-    filterUpdated: Ember.observer('filter', function() {
-        let filter = this.get('filter');
-        if (filter) {
-            let key = this.get('key');
-            let { start, end } = invertDateRangeFilter(key, filter);
+    statePrevious: [],
+    // this isn't getting triggered
```

Comment:
Does it trigger if it depends only on `state`? Observing a property on an object might only work if it's set like `state.set('start')`, though I would have thought setting `state` to a new object would trigger it, too.